aws/src/main/java/org/apache/iceberg/aws/AwsClientFactories.java

```diff
@@ -50,6 +50,14 @@ public static AwsClientFactory defaultFactory() {
   public static AwsClientFactory from(Map<String, String> properties) {
     String factoryImpl = PropertyUtil.propertyAsString(
         properties, AwsProperties.CLIENT_FACTORY, DefaultAwsClientFactory.class.getName());
+    if (PropertyUtil.propertyAsBoolean(properties,
```

Comment:
nit: for multi-line code, prefer variables to be indented at the same level for clarity, for example:

```
PropertyUtil.propertyAsBoolean(
  properties, AwsProperties.GLUE_LAKEFORMATION_ENABLED, AwsProperties.GLUE_LAKEFORMATION_ENABLED_DEFAULT)
```

or 

```
PropertyUtil.propertyAsBoolean(
  properties, 
  AwsProperties.GLUE_LAKEFORMATION_ENABLED, 
  AwsProperties.GLUE_LAKEFORMATION_ENABLED_DEFAULT)
```